src/app/media-viewer-wrapper/media-viewer-wrapper.component.html

```diff
@@ -81,27 +78,27 @@ <h2 class="govuk-heading-m  govuk-!-margin-top-4">Media Viewer Demo</h2>
                          [comments]="comments"></mv-comments-summary>
     <section [id]="selectedTab">
       <div *ngIf="selectedTab === 'pdf'">
-        <mv-media-viewer [url]="pdfUrl"
+        <mv-media-viewer [url]="url"
                          [downloadFileName]="pdfFileName"
                          [toolbarButtonToggles]="toolbarButtons"
                          [contentType]="'pdf'"
-                         [annotationSet]="pdfAnnotationSet"
+                         [annotationSet]="annotationSet"
                          [showCommentSummary]="showCommentSummary"
                          [showAnnotations]="showAnnotations"
                          [showToolbar]="showToolbar"></mv-media-viewer>
       </div>
       <div *ngIf="selectedTab === 'image'">
-        <mv-media-viewer [url]="imageUrl"
+        <mv-media-viewer [url]="url"
                          [downloadFileName]="imageFileName"
```

Comment:
do we actually need all three instances now, since we're changing the url/annotationSet values dynamically?

Reply:
good question

Reply:
not needed any more so I removed them